spec/BasicSpec.js

```diff
@@ -870,6 +870,52 @@ describe('HtmlWebpackPlugin', function () {
     });
   });
 
+  it('allows to modify sequentially the html during html-webpack-plugin-after-html-processing event', function (done) {
```

Comment:
This test is just perfect - can we copy it for your usecase where you don't want to modify the root object?
And one where we mix both?